tests/security_features_test.go

```diff
@@ -239,7 +222,7 @@ var _ = Describe("[Serial][sig-compute]SecurityFeatures", Serial, decorators.Sig
 	Context("Check virt-launcher capabilities", func() {
 		var container k8sv1.Container
 
-		It("[test_id:4300]has precisely the documented extra capabilities relative to a regular user pod", func() {
+		It("[test_id:4300]has precisely the documented extra capabilities relative to a regular user pod", decorators.Conformance, func() {
```

Comment:
I do wonder if the benefit of keeping this case and `[test_id:2953][test_id:2895]` case separate is worth it. I would probably generalize it down to "virt-launcher Pod has least-privileged security settings" and have all relevant assert in one test case.

Reply:
To me, it's worth it because of parallelism.
I mean, test `[test_id:2953][test_id:2895]` requires to be serial since they are changing the kv config.
Merging them will result in the test being serial, hence out of the conformance.
I will, anyway, add a commit to promote parallel tests that don't require to be serial.
WDYT?

Reply:
>I will, anyway, add a commit to promote parallel tests that don't require to be serial. 

+100

ah, I am missing some more comprehensive test. 4300 is just about capabilities. Also imho we should drop the root case, that is now alive only because we had request to allow more time to transition but the coverage we have is barely enough :)

Reply:
I generally agree, but I would leave it as is for the moment and leave this effort for when we discontinue the Root FG. Isn't that the plan?